modules/apps/users-admin/users-admin-web/src/main/resources/META-INF/resources/css/main.scss

```diff
@@ -30,7 +30,7 @@
 	}
 
 	.hide-data-labels .toggle-switch-handle:after {
-		content: normal !important;
+		content: none !important;
```

Comment:
Hey @georgel-pop-lr, why this change was necessary? Sorry for the inconvenience here :P

Reply:
Hi @diegonvs, is very similar with the solution @pat270 implemented inside clay https://github.com/liferay/clay/pull/4686/files

Basically, is needed for OAW tool to not detect this as a content added from CSS styles, as this content is considered not accessible by reader tools.

Regards.

Reply:
Oh ok!